server/middleware/authMiddleware.js

```diff
@@ -1,19 +1,22 @@
-const jwt = require('jsonwebtoken');
+const jwt = require("jsonwebtoken");
+const { TOKEN_SECRET_KEY } = require("../config");
 
 const requireAuth = (req, res, next) => {
   const token = req.cookies.jwt;
 
   if (token) {
-    jwt.verify(token, '', (err, decodedToken) => {
+    jwt.verify(token, TOKEN_SECRET_KEY, (err, decodedToken) => {
       if (err) {
         // token expired
       } else {
         // user is authenticated
+        req.id = decodedToken.id; // get current user id

```

Comment:
can we make this a bit more specific - so something like userId, so we know what we are talking about?

Reply:
Changed it to userToken and accessing userToken.id